pyfolio/timeseries.py

```diff
@@ -600,31 +493,19 @@ def cum_returns(returns, starting_value=None):
     where it is possible to sum instead of multiplying.
     """
 
-    # df_price.pct_change() adds a nan in first position, we can use
-    # that to have cum_returns start at the origin so that
-    # df_cum.iloc[0] == starting_value
-    # Note that we can't add that ourselves as we don't know which dt
-    # to use.
-    if pd.isnull(returns.iloc[0]):
-        returns.iloc[0] = 0.
+    starting = 0 if starting_value is None else starting_value
```

Comment:
Can we simplify this by just passing `starting_value=0` as the default argument, and removing this line? IMO this pattern is generally more useful if the default argument is mutable, eg if you had `starting_value=[]`, but in this case we can just pass `starting_value=0` as the default.